tutorials/autotvm/tune_simple_template.py

```diff
@@ -32,6 +32,8 @@
 # Install dependencies
 # --------------------
 # To use autotvm package in TVM, we need to install some extra dependencies.
+# This step is needed if you decide to use xgboost for tuning. This example
+# as it is doesn't need xgboost.
```

Comment:
This sentense looks a bit weird to me. Are you saying something like "This step can be skipped as it doesn't need XGBoost."?

Reply:
Yes thank you, I'll use your wording :).